indico/modules/events/abstracts/__init__.py

```diff
@@ -46,6 +46,7 @@ def _clear_boa_cache(sender, obj=None, **kwargs):
     clear_boa_cache(event)
 
 
+@signals.menu.items.connect_via('event-editing-sidemenu')
```

Comment:
The CfA link in the hybrid view shouldn't point to the management page but rather the reviewing area, i.e. in a new function that uses the same logic used in `_extend_event_menu`.

Similar for the CfP link.

Reply:
MenuEntryData contains an endpoint not a url (like the management menu classes). Should I create a custom field that serializes the endpoint to a url in the schema or is using the endpoint with a url value acceptable here?

Reply:
But that's just for the conference menu, keep using `SideMenuItem` of course - my comment was just about the visibility logic. Sorry if that wasn't clear.